bin/jsondiff.py

```diff
@@ -3,6 +3,18 @@
 import argparse
 import diff
 import deepdiff
+import logging
+
+from mooseutils.validate import validate_extension, validate_files_exist
+
+# Configure logging to overwrite the log file for each run, add option: filemode='w'
+logging.basicConfig(filename='MOOSETools.log',
+                    level=logging.INFO,
+                    format='%(asctime)s %(levelname)s %(message)s',
+                    filemode='w',
+                    datefmt='%m/%d/%Y %H:%M:%S')
+
+logging.info('Application started. Logging to file MOOSETools.log')
```

Comment:
```suggestion
```
I always just log to the screen. In the future we might setup logging to a file, but it will be controlled by an environment variable. You want people to see messages and warnings without looking in a file.